### ISSUE TITLE
Unmet peer dependency warnings for linter (eslint-config-airbnb@15.0.2)

### ISSUE THREAD

**Issue by zxul767** [wizeline/wizelink-front#17]
I got this error while running the `install.sh` script. Is this a spurious error we can ignore or will it actually affect us when running the linter?

```
Installing dev dependencies...
npm WARN eslint-config-airbnb@15.0.2 requires a peer of eslint-plugin-jsx-a11y@^5.1.1 but none was installed.
npm WARN eslint-config-airbnb@15.0.2 requires a peer of eslint-plugin-import@^2.6.1 but none was installed.
Installing dependencies...
wizelink-front@0.1.0 /Users/zxul767/src/wizeline/side-projects/wizelink-front
├── UNMET PEER DEPENDENCY eslint-plugin-import@2.2.0
├── UNMET PEER DEPENDENCY eslint-plugin-jsx-a11y@5.0.3
└─┬ redux-logger@3.0.6
  └── deep-diff@0.3.8

npm WARN eslint-config-airbnb@15.0.2 requires a peer of eslint-plugin-jsx-a11y@^5.1.1 but none was installed.
npm WARN eslint-config-airbnb@15.0.2 requires a peer of eslint-plugin-import@^2.6.1 but none was installed.
```

**Comment by vusause** [comment]
I've found no issue with this as of yet, but this should be a simple fix by updating the package versions in the `package.json`

**Comment by zxul767** [comment]
Fixed by #19
